### ISSUE TITLE
Julia 0.5.0 deprication notice

### ISSUE THREAD

**Issue by GeoHubX** [JuliaLang/IJulia.jl#474]
I have been using IJulia happily for a while. With the changes induced through the update to Julia 0.5.0 there are now warnings when invoking Pkg.build("IJulia").
While it occurs when calling PyPlot in Jupyter, the error message says:

"WARNING: Base.writemime is deprecated.
  likely near C:\Users\. . . \.julia\v0.5\IJulia\src\kernel.jl:31" 
many times.

While I expect you have seen this, I just thought I'd pass it along, hoping to identify a problem needing resolution.
Thanks.

**Comment by yuyichao** [comment]
Which IJulia version are you using? This should have been fixed already

**Comment by yuyichao** [comment]
Actually ref https://github.com/JuliaLang/IJulia.jl/issues/426#issuecomment-242544318, should be a PyPlot issue that's probably just need a new tag.

**Comment by GeoHubX** [comment]
Thanks for the reference to #426. 

I had been using IJulia 1.3.2.

I applied the Pkg.checkout(“Pyplot”) fix, but still got the warning:

"WARNING: Base.writemime is deprecated.
  likely near C:\Users\testy\.julia\v0.5\IJulia\src\kernel.jl:31" without the reference to PyPlot.

It only showed up twice, though.

When I tried Pkg.checkout("IJulia") to see if the master would help, but I just got more of the same, only more, referring to PyCall as well as IJulia, so I reverted.

I then tried Pkg.checkout("PyCall"), which took care of it. I will append to the other conversation as well. 

I am not trying to drag this chain off track, just close the issue from my side as well, with more info than at #426.
